chapters/features.txt

```diff
@@ -3489,6 +3492,10 @@ All Vulkan graphics implementations must: support the following features:
   * <<features-robustBufferAccess, pname:robustBufferAccess>>
 ifdef::VK_VERSION_1_1[]
   * <<features-multiview, pname:multiview>>, if Vulkan 1.1 is supported.
+ifdef::VK_KHR_shader_draw_parameters[]
```

Comment:
if you have a 1.0 implementation there is no `shaderDrawParameters` to expose

Reply:
ergo the `ifdef::VK_VERSION_1_1[]`.